01.fizzbuzz/fizzbuzz.js

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env node
+'use strict'
+
+class FizzBuzz {
+  static exec (num) {
+    let result = ''
+
+    if (FizzBuzz.#isFizz(num)) {
```

Comment:
`#isFizz` はここでしか使っていないのでメソッドに切り出さなくても

```suggestion
    if (num % 3 === 0) {
```

でも良かったかなと思いました。`#isBuzz` の方も同様ですね。